recipes/gromacs/2020/build.sh

```diff
@@ -41,23 +41,25 @@ ${CXX} -O3 -mavx512f -std=c++11 \
 -o ${PREFIX}/bin.AVX_512/identifyavx512fmaunits \
 ${SRC_DIR}/src/gromacs/hardware/identifyavx512fmaunits.cpp
 
+cp -a ${RECIPE_DIR}/gmx-chooser.bash ${PREFIX}/bin/gmx
+chmod a+x ${PREFIX}/bin/gmx
 
 # Create wrapper and activation scripts
 # Variable declaration from MPI script fewer changes if left in.
 
 gmx='gmx'
-
+￼
 mkdir -p "${PREFIX}/etc/conda/activate.d"
 touch "${PREFIX}/bin/${gmx}"
 chmod +x "${PREFIX}/bin/${gmx}"
-
+￼
```

Comment:
More

Reply:
I think this is needed.   We could swap the order so that the 'cat' to the file is done first, and then the permissions are set, but I think this is fine (just not the tidiest).

Reply:
I'm fine with a blank line, but it shouldn't have some weird character

Reply:
This was an accident, and should be gone now.